LocalyticsXamarin/LocalyticsXamarin.Android/Transforms/Metadata.xml

```diff
@@ -136,11 +139,6 @@
      <attr path="/api/package[@name='com.localytics.android']/interface[@name='AnalyticsListener']/method[@name='localyticsSessionWillClose']" name="managedName">SessionWillClose</attr>
      <attr path="/api/package[@name='com.localytics.android']/interface[@name='AnalyticsListener']/method[@name='localyticsDidTagEvent']" name="managedName">DidTagEvent</attr>
      <attr path="/api/package[@name='com.localytics.android']/interface[@name='AnalyticsListener']/method[@name='localyticsSessionWillOpen']" name="managedName">SessionWillOpen</attr>
-
```

Comment:
If it's not an issue, leave this in place. it helps with attribute name changing and would be used when JNI generation is fixed.